common.gypi

```diff
@@ -407,9 +407,15 @@
         'libraries': [ '-lelf' ],
       }],
       ['OS=="freebsd"', {
-        # Use this flag because on FreeBSD std::pairs copy constructor is non-trivial
-        # https://lists.freebsd.org/pipermail/freebsd-toolchain/2016-March/002094.html
-        'cflags': [ '-D_LIBCPP_TRIVIAL_PAIR_COPY_CTOR=1' ],
+        'conditions': [
+          ['llvm_version < "4.0"', {
```

Comment:
Is this check future proof? Not sure how the `<` operator operates, but what happens if `llvm_version == "10.0"`? If it is a string comparison, `'10.0' < '4.0'`.